ytypes/node_test.go

```diff
@@ -1101,11 +1125,68 @@ func TestGetNode(t *testing.T) {
 			},
 		},
 		inPath: mustPath("/multilist[keyone=1][keytwo=2]"),
+		inArgs: []GetNodeOpt{&GetHandleWildcards{}},
```

Comment:
This looks like it changes this test case to test a different behaviour - i.e., with the wildcard handling enabled or not -- can we have two tests:
1. Explicit key match with wildcard handling off
2. Explicit key match with wildcarding handling on